framework/components/ecl-site-headers/ecl-site-headers.twig

```diff
@@ -38,23 +42,23 @@
 {# Print result #}
 <header class="{{ _css_class }}" {{ _extra_attributes }} role="banner">
 
-  {% if links is defined %}
+  {% if user_menu.links is defined %}
```

Comment:
Beware of this (the same applies to `{% if site_switcher.political is defined and site_switcher.info is defined %}`): if `user_menu` is not defined, twig will throw errors